osu.Game.Rulesets.Mania/ManiaRuleset.cs

```diff
@@ -427,6 +427,20 @@ public override IRulesetFilterCriteria CreateRulesetFilterCriteria()
         public override RulesetSetupSection CreateEditorSetupSection() => new ManiaSetupSection();
 
         public override DifficultySection CreateEditorDifficultySection() => new ManiaDifficultySection();
+
+        // Mania doesn't have rate-adjusted attributes anymore?
+
+        //public override BeatmapDifficulty GetRateAdjustedDifficulty(IBeatmapDifficultyInfo baseDifficulty, double rate)
+        //{
+        //    BeatmapDifficulty adjustedDifficulty = new BeatmapDifficulty(baseDifficulty);
+
+        //    double hitwindow = 64.0 - 3 * adjustedDifficulty.OverallDifficulty;
+        //    hitwindow /= rate;
+        //    adjustedDifficulty.OverallDifficulty = (float)(64.0 - hitwindow) / 3;
+
+        //    return adjustedDifficulty ?? (BeatmapDifficulty)baseDifficulty;
+        //}
+
```

Comment:
Shouldn't exist.

Reply:
okay, I've just wanted to overask about it

Reply:
fixed